org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -8,14 +8,13 @@
 var request = require('request-promise');
 var OpenT2T = require('opent2t').OpenT2T;
 var Crypto = require('crypto');
-var accessTokenInfo = require('./common').accessTokenInfo;
 
 /**
 * This translator class implements the "Hub" interface.
 */
 class Translator {
-    constructor(accessToken) {
-        this._accessToken = accessToken;
+    constructor(authTokens) {
+        this._authTokens = authTokens;
```

Comment:
This would break hubcontroller + hubcontroller tests + CLI -- are those changs also coming in?